src/AWSS3.jl

```diff
@@ -115,7 +116,7 @@ s3_get(a...; b...) = s3_get(global_aws_config(), a...; b...)
 Like `s3_get` but streams result directly to `filename`.  Keyword arguments accept are
 the same as those for `s3_get`.
 """
-function s3_get_file(aws::AbstractAWSConfig, bucket, path, filename; version="", kwargs...)
+function s3_get_file(aws::AbstractAWSConfig, bucket, path, filename; version::S3PathVersion=nothing, kwargs...)
     stream = s3_get(aws, bucket, path; return_stream=true, kwargs...)
```

Comment:
Currently this function ignores `version` entirely:
```suggestion
function s3_get_file(aws::AbstractAWSConfig, bucket, path, filename; version::S3PathVersion=nothing, kwargs...)
    stream = s3_get(aws, bucket, path; return_stream=true; version=version, kwargs...)
```
Should add a test for this using a versioned file

Reply:
yikes! added this (and a test)...